recipes-connectivity/network-manager-applet/network-manager-applet-1.4.4/disable-auto-ethernet.patch

```diff
@@ -33,12 +33,33 @@
  	return TRUE;
  }
  
-@@ -238,8 +218,6 @@ ethernet_add_menu_item (NMDevice *device
+@@ -118,8 +98,6 @@ ethernet_add_menu_item (NMDevice *device
  
- 		if (g_slist_length (connections))
- 			add_connection_items (device, connections, carrier, active, ADD_INACTIVE, menu, applet);
+ 		if (connections->len)
+ 			applet_add_connection_items (device, connections, carrier, active, NMA_ADD_INACTIVE, menu, applet);
 -		else
--			add_default_connection_item (device, carrier, menu, applet);
+-			applet_add_default_connection_item (device, DEFAULT_ETHERNET_NAME, carrier, menu, applet);
  	}
+ }
+ 
+--- a/src/applet.c
++++ b/src/applet.c
+@@ -1354,8 +1354,7 @@ nma_menu_device_get_menu_item (NMDevice
+ 	}
+ 
+ 	if (!managed) {
+-		item = gtk_menu_item_new_with_label (_("device not managed"));
+-		gtk_widget_set_sensitive (item, FALSE);
++		return NULL;
+ 	}
+ 
+ 	return item;
+@@ -1372,7 +1371,7 @@ add_device_items (NMDeviceType type, con
+ 	for (i = 0; all_devices && (i < all_devices->len); i++) {
+ 		NMDevice *device = all_devices->pdata[i];
  
- 	g_slist_free (connections);
+-		if (nm_device_get_device_type (device) == type) {
++		if (nm_device_get_device_type (device) == type && nm_device_get_state (device) != NM_DEVICE_STATE_UNMANAGED ) {
```

Comment:
<nit> wrap the test condition after the `&&` to keep from excessively exceeding 80 cols</nit>